vhdbuilder/packer/install-dependencies.sh

```diff
@@ -409,7 +409,7 @@ TIGERA_OPERATOR_IMAGES="
 1.13.5
 "
 for TIGERA_OPERATOR_IMAGE in ${TIGERA_OPERATOR_IMAGES}; do
-    CONTAINER_IMAGE="mcr.microsoft.com/oss/tigera/kube-controllers:v${TIGERA_OPERATOR_IMAGE}"
+    CONTAINER_IMAGE="mcr.microsoft.com/oss/tigera/operator:v${TIGERA_OPERATOR_IMAGE}"
```

Comment:
can you also add this line to the file:
https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/test/linux-vhd-content-test.sh
so that next time we don't fail this issue?
  {
    "downloadURL": "mcr.microsoft.com/oss/tigera/operator:v*",
    "versions": ["1.13.5"]
  },